### ISSUE TITLE
Video posting returns ok status but uploaded videos don't show up in instagram

### ISSUE THREAD

**Issue by whicher** [ohld/igbot#1211]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- You will be asked some questions and requested to provide some information, please read them **carefully** and answer completely.
- Put an `x` into all the boxes [ ] relevant to your issue (like so [x]).
- Use the *Preview* tab to see how your issue will actually look like.

---

### Before submitting an issue, make sure you have:
- [ x] Updated to the latest version
- [ x] Read the [README](https://github.com/instagrambot/instabot/blob/master/README.md) and [docs](https://github.com/instagrambot/instabot/tree/master/docs)
- [ x] [Searched](https://github.com/instagrambot/instabot/search?type=Issues) the bugtracker for similar issues including **closed** ones

### Purpose of your issue?
- [x ] Bug report (encountered problems/errors)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections requests more details for particular types of issues, you can remove any section (the contents between the triple ---) not applicable to your issue.

---

### For a *bug report*, you **must** include the Python version used, *code* that will reproduce the error, and the *error log/traceback*.

Paste the output of ``python -V`` here:
3.6.9

Code:
From examples/video/

```python
# Example code that will produce the error reported
from instabot import Bot

bot = Bot()
bot.login(username="user", password="pass")
os.path.dirname(os.path.realpath(__file__)) + "/media/" + video, caption=caption):
```

Error/Debug Log:

```python
{"xsharing_nonces": {}, "status": "ok"}
```

---

### Describe your issue

A little bit similar to #1135. Return status is ok from the api however uploaded video doesn't show up in the profile. Photo seems working fine though.

**Comment by bruvv** [comment]
Try again within 5 - 10 days probably blocked by instagram. Instagram is doing this stealthy so it is very difficult to find the error.

**Comment by bruvv** [comment]
I am closing this in favor of this post: https://github.com/instagrambot/instabot/issues/1194 please continu the ban discussion there Thanks for the issue btw!
